src/@openeventkit/event-site/templates/marketing-page-template.js

```diff
@@ -149,33 +150,53 @@ const MarketingPageTemplate = ({
             }
           }}
         >
-          {hero.backgroundVideo &&
-          <Box
-            component="video"
-            preload="auto"
-            playsInline
-            autoPlay
-            muted
-            loop
-            sx={{
-              position: "absolute",
-              margin: "auto",
-              inset: 0,
-              top: {
-                xs: "-48%",
-                md: "-22%",
-                xl: "-28%"
-              },
-              left: {
-                sm: 16,
-                md: 24,
-                lg: 32,
-                xl: 40
-              }
-            }}
-          >
-            <source src={hero.backgroundVideo.publicURL} type="video/mp4"/>
-          </Box>
+          {hero.backgroundMedia &&
+          <>
+            {getMediaFileExtension(hero.backgroundMedia.publicURL) === "video" ?
```

Comment:
Since you are comparing to "video", shouldn't function be getMediaFileType?
Also, you could export in utils an enum like object as
const MEDIA_TYPES = Object.freeze({
  VIDEO: 'video',
  IMAGE: 'image'
});